manageability-docker/README.md

```diff
@@ -13,11 +13,20 @@ This folder contains the build instructions for creating mangeability docker con
 ## HOW TO BUILD INB IMAGE AND START THE CONTAINER
 
 * Unzip the package(inb_azure_container.zip/inb_tb_container.zip).
-* If planning to use the sample_customer_mqtt_client provided, open the cloud_source file and uncomment the lines specified in the file.
+* If planning to use the `sample_customer_mqtt_client` provided please refer the section below. 
 * If using Thingsboard, edit the tb_conf_file and fill in the respective Thingsboard server details and device tokens along with tls certificate of thingsboard server if TLS is enabled on the Thingsboard server. If using TLS for Thingsboard uncomment the line 'cp /src/thingsboard.pub.pem /etc/intel-manageability/secret/cloudadapter-agent/thingsboard.pub.pem' in the cloud_source file.
```

Comment:
The grammar is a little off.  

Perhaps.  "If using the provided sample, 'sample_customer_mqtt_client', please refer to the section below.  And then maybe link 'section below' to that section.